test/functional/config.js

```diff
@@ -64,7 +64,10 @@ testingEnvironments[testingEnvironmentNames.mobileBrowsers] = {
         accessKey: process.env.BROWSER_STACK_ACCESS_KEY
     },
 
-    retryTestPages: true,
+    ssl: {
```

Comment:
If you will implement some other changes, could you please remove the `ssl` and `retryTestPages` from mobile browsers and macOS + Edge config? 

SSL is quite useless without enabling `retryTestPages`, but we can't enable it right now due to workers are not supported by the current Safari and iOS test devices and we need to debug some complex tests to use newer browsers versions.